src/mtype.d

```diff
@@ -250,7 +250,8 @@ extern (C++) Type stripDefaultArgs(Type t)
                             for (size_t j = 0; j < params.dim; j++)
                                 (*params)[j] = (*parameters)[j];
                         }
-                        (*params)[i] = new Parameter(p.storageClass, ta, null, null);
+                        StorageClass stc = p.storageClass & (~STCauto); // issue 14656
+                        (*params)[i] = new Parameter(stc, ta, null, null);
```

Comment:
Why do we have 2 different fixes in master and stable?
https://github.com/D-Programming-Language/dmd/pull/5032
https://github.com/D-Programming-Language/dmd/pull/4728